### ISSUE TITLE
component loader feeders

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8489]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

From https://github.com/ArctosDB/dev/issues/170 - I'm in the process of turning one old-style "comboform" (identifications and part container movement) into two new style simple tools.

I said "[simple good](https://github.com/ArctosDB/dev/issues/170#issuecomment-2624739314)" - more single-task forms are less likely to result in some sort of confused user doing some regrettable thing.

@DerekSikes says "[clicks bad](https://github.com/ArctosDB/dev/issues/170#issuecomment-2624988287)" - efficiency is important, risk/complexity is manageable.

I think this is about 100% a social issue, so I'm bringing it back over here for discussion. I'm sure we're both right - people do get lost in more complexity (but also in more things, and I've made one more than before), and a user on a specific task can benefit from a fewer-clicks workflow (probably also in fewer chances to oopsie something). I don't have much of a sense of how to balance those things, or how sturdy the guardrails should be around some potential cliff. (The rocks at the bottom of this one don't see **too** sharp and jagged, it's mostly doing things that can be undone or at least understood, but I'm not sure how accurate that is or how far it scales.) 

These forms are (so far) pretty simple so it's not much of a problem to change the direction, behavior, or about anything else as long as we don't get too far from 'munge search results into component loader.' It would be good to have a bit of a direction before this goes too far. Should we aim for something like one tool per task, support multi-task specialized tools, possibly just allow the menu to open multiple tools at once, ?????????????????????

HELP!

**Comment by dustymc** [comment]
See https://github.com/ArctosDB/dev/issues/170#issuecomment-2626244760 - @DerekSikes idea of chaining tools together via post-success links seems elegant and unobtrusive.

**Comment by mkoo** [comment]
overall I favor simpler the better in the sense of one tool per task. however I agree this is 100% social decision with a huge helping of practicality. Some sets of related tasks are infrequent done so might be easier to maintain a single multitool for tasks? Or maybe I have that backward: frequently done related tasks should have a single swiss armyknife tool? let's gather comments and examples and may need to become an AWG topic

**Comment by campmlc** [comment]
Just curious is this will be an AWG topic? I agree with gathering comments and examples.
